### PR TITLE
Week Highlighting

### PR DESCRIPTION
**Pull request by JKercheval**
Pay closest attention to FSCalendar.h and FSCalendar.m

### CODE REVIEW

File: Example/CalendarIdentifierViewController.m [hunk 1]
```diff
@@ -32,6 +32,7 @@ - (void)loadView
     FSCalendar *calendar = [[FSCalendar alloc] initWithFrame:CGRectMake(0, 64, CGRectGetWidth(self.view.frame), height)];
     calendar.dataSource = self;
     calendar.delegate = self;
+  calendar.showsPlaceholders = YES;
```
Comment (kosukeyano) [inline Example/CalendarIdentifierViewController.m]: I think the indentation is supposed to be 4 spaces here.

File: Example/CalendarIdentifierViewController.m [hunk 2]
```diff
@@ -93,4 +99,17 @@ - (void)pickerView:(UIPickerView *)pickerView didSelectRow:(NSInteger)row inComp
     _calendar.identifier = identifier;
 }
 
+- (BOOL) calendar:(FSCalendar *)calendar isEnabledWeekHightlight:(NSDate *)date
+{
+  if ([date timeIntervalSince1970] >= [[NSDate date] timeIntervalSince1970]) {
```
Comment (kosukeyano) [inline Example/CalendarIdentifierViewController.m]: The if statement is unnecessary here.

File: FSCalendar/FSCalendar.h [hunk 3]
```diff
@@ -240,6 +270,14 @@ IB_DESIGNABLE
 @property (strong, nonatomic) NSDate *currentPage;
 
 /**
+ *  The Date of the first day of the week that correcsponds to the highlighted week
```
Comment (kosukeyano) [inline FSCalendar/FSCalendar.h]: You have a typo here in the doc strings: corresponds*

File: FSCalendar/FSCalendar.h [hunk 4]
```diff
@@ -240,6 +270,14 @@ IB_DESIGNABLE
...
+ *
+ * @desc In Month mode, you can have the week of the currently selected day hightlghted, if you do then this
```
Comment (kosukeyano) [inline FSCalendar/FSCalendar.h]: You have a typo here in the doc strings: highlighted*

**Comment by kosukeyano** [comment]
+1

**Comment by JKercheval** [comment]
Updated code for all comments.

**Comment by jeffChavez** [comment]
Looks good

**Pull request merged**